Adding Images and GIFs to readme (#71)
LibreTexts/ckeditor-binder-plugin: README.md

A CKeditor plugin that makes adding binder enabled pre tags easy.

## Installation

```
yarn install
```

## Start Dev Server

```
yarn start
```

## Build Prod Version

```
yarn build
```

## Linter

This repo follows Airbnb Javascript guide, see [here](https://github.com/airbnb/javascript) for more information.

## Usage with LibreTexts

CKEditor is a rich text editor which enables the user to write content directly inside web pages or online applications. Through the Binder Plugin, authors of textbooks have the option to include code within in their textbook pages to create a more interactive experience for their students.

Currently the plugin supports Python 3, Julia, R, Octave, and SageMath. Additionally, all languages with the exception of SageMath support syntax highlighting for an easier scripting experience. 

### Step-by-Step Process to use CKEditor Binder Plugin

#### Creating a new page
Authors have the option to add code to a new page.

![](tutorialVisuals/newPage.gif)

> #### Editing a exisiting page

Reviewer: ```suggestion
#### Editing a existing page
```

Additionally, authors can also add code to pages that they have already created. To do so, simply click on the "Edit" button.
![](tutorialVisuals/editOld.gif)

> #### Locating the Plugin and Adding Code

Reviewer: ```suggestion
#### Locating the plugin and adding code
```
Capitalization consistency, since the first two level-4 headers were in sentence case.

> The Binnder Logo is the last button on the navigation bar of the text editor.  

Reviewer: ```suggestion
The Binder logo is the last button on the navigation bar of the text editor.  
```

![](tutorialVisuals/ckePlugin.png)  

> #### Add Code

Reviewer: ```suggestion
#### Add code
```
Capitalization consistency, since the first two level-4 headers were in sentence case.

> To add a block of code, select the language and add code in the text space. Remember to press run and allow the ouput to fully apear in order for it to properly show up on the webpage.

Reviewer: ```suggestion
To add a block of code, select the language and add code in the text space. Remember to press run and allow the output to fully appear in order for it to properly show up on the webpage.
```

> To add a block of code, select the language, and add code in the text space. Remember to press run, and allow the ouput to fully apear in order for it to properly show up on the webpage

Reviewer 2: I believe you don't need a comma before both `and`. Also, appear.

Reviewer 2: And a period in the end is missing.

![](tutorialVisuals/addCode.gif) 

### An Example Page

> Remeber to press save once edits to the page are done, the following is an example page utilizing the Binder Plugin. As seen, authors can use libraries and packages already available.

Reviewer: ```suggestion
Remember to press "Save" once edits to the page are done, the following is an example page utilizing the Binder Plugin. As seen, authors can use libraries and packages already available.
```
Capitalization for consistency.

> ![](tutorialVisuals/examplePage.gif)

Reviewer 2: Somehow the image and the text position is messed up. Can you try to fix it?

Author: I'm not entirely sure what you mean here. Do you mean that they positions of the two GIFS look different in the below screenshot?
![image](https://user-images.githubusercontent.com/53124295/78321592-4a59ef80-7521-11ea-9afe-b3e18ac5fbc4.png)
